scenes/src/app/scene-overlay-container.ts

```diff
@@ -0,0 +1,11 @@
+import {OverlayContainer} from '@angular/cdk/overlay';
+
+export class SceneOverlayContainer extends OverlayContainer {
+  _createContainer(): void {
+    const container = document.createElement('div');
+    container.classList.add('scene-overlay-container');
+
+    document.querySelector('#scene-content-container')?.appendChild(container);
+    this._containerElement = container;
```

Comment:
```suggestion
    const container = this._document.createElement('div');
    container.classList.add('scene-overlay-container');

    this._document.querySelector('#scene-content-container')?.appendChild(container);
    this._containerElement = container;
```
I think you can use `this._document` from the [OverlayContainer](https://cs.opensource.google/angular/components/+/master:src/cdk/overlay/overlay-container.ts;l=31;drc=29eec7729253ef8329267dc21eb604eacdf15c7f)